src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const empoyeeslist = document.querySelector('ul');
```

Comment:
There is a typo in the variable name `empoyeeslist`. It should be `employeesList` to reflect the correct spelling.